src/scss/utilities/_grid.scss

```diff
@@ -121,7 +125,7 @@
 .grid__col {
   display: inline-block;
   box-sizing: border-box;
-  flex: 0 0 auto;
+  // flex: 0 0 auto;
```

Comment:
Probably just want to remove this rather than comment it out

Reply:
Agreed.